---
title: "mr_traitohol_analysis"
author: "Shea Andrews"
date: "25/03/2018"
output:
  html_document:
    df_print: paged
    toc: true
    number_sections: false
    toc_depth: 4
    toc_float:
      collapsed: false
      smooth_scroll: false
editor_options: 
  chunk_output_type: console
params:
  trait_load.path: 
  trait_aaos.path: 
  trait_ab42.path: 
  trait_ptau.path: 
  trait_tau.path: 
  trait_hipv.path: 
  out.path: 
  trait.name: 
  trait.code: 
  p.threshold: 
---

```{r setup analysis, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

library(tidyverse)   ## For data wrangling 
library(Hmisc)       ## Contains miscillaneous funtions 
library(TwoSampleMR) ## For conducting MR https://mrcieu.github.io/TwoSampleMR/ 
library(ggplot2)     ## For plotting 
library(MRPRESSO)    ## For pleiotropy analysis  
```

```{r Data Input, include=FALSE, cache=TRUE, echo=FALSE}
### ===== Load Harmonized Data ===== ###

## ---- Trait ---- ##
trait_LOAD.MRdat <- read_csv(params$trait_load.path)
trait_AAOS.MRdat <- read_csv(params$trait_aaos.path)
trait_AB42.MRdat <- read_csv(params$trait_ab42.path)
trait_Ptau.MRdat <- read_csv(params$trait_ptau.path)
trait_Tau.MRdat <- read_csv(params$trait_tau.path)
trait_hipv.MRdat <- read_csv(params$trait_hipv.path)

## Data Frame of nsnps and number of iterations 
df.NbD <- data.frame(n = c(10, 50, 100, 500, 1000, 1500, 2000),
       NbDistribution = c(1000, 5000, 10000, 25000, 50000, 75000, 100000))

nsnps <- max(nrow(trait_LOAD.MRdat), nrow(trait_AAOS.MRdat), nrow(trait_AB42.MRdat), nrow(trait_Ptau.MRdat), nrow(trait_Tau.MRdat), nrow(trait_hipv.MRdat))
NbDistribution <- df.NbD[which.min(abs(df.NbD$n - nsnps)), 2]

```


## Pleiotropy 
Pleiotropy was assesed using Mendelian Randomization Pleiotropy RESidual Sum and Outlier (MR-PRESSO), that allows for the evlation of evaluation of horizontal pleiotropy in a standared MR model. MR-PRESSO performs a global test for detection of horizontal pleiotropy and correction of pleiotropy via outlier removal. 
<br> 

```{r pleiotropy load, echo = F, cache=T}
## ---- LOAD ---- ##
trait_LOAD_presso <- mr_presso(BetaOutcome = "beta.outcome", 
          BetaExposure = "beta.exposure", 
          SdOutcome = "se.outcome", 
          SdExposure = "se.exposure", 
          OUTLIERtest = TRUE, 
          DISTORTIONtest = TRUE, 
          data = as.data.frame(trait_LOAD.MRdat), 
          NbDistribution = NbDistribution,  
          SignifThreshold = 0.05)

if("Global Test" %in% names(trait_LOAD_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_LOAD_presso$`MR-PRESSO results`)){
  if(is.character(trait_LOAD_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    load_snps <- trait_LOAD_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
  load_snps <- trait_LOAD.MRdat[trait_LOAD_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
  trait_LOAD.MRdat <- trait_LOAD.MRdat[-trait_LOAD_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_LOAD_presso$`MR-PRESSO results`)){
  load_presso <- trait_LOAD_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
    load_presso <- trait_LOAD_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ LOAD:** The MR-PRESSO global test for pleiotropy was `r if(load_presso < 0.05){paste('significant', ' (p = ', load_presso, ').', sep = "")}else{paste('non-significant')}` `r if(load_presso < 0.05 & any(ls() %in% "load_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(load_snps, collapse=", ") )}`
<br> 

```{r pleiotropy aaos, echo = F, cache=T}
## ---- AAOS ---- ##
trait_AAOS_presso <- mr_presso(BetaOutcome = "beta.outcome", 
          BetaExposure = "beta.exposure", 
          SdOutcome = "se.outcome", 
          SdExposure = "se.exposure", 
          OUTLIERtest = TRUE, 
          DISTORTIONtest = TRUE, 
          data = as.data.frame(trait_AAOS.MRdat), 
          NbDistribution = NbDistribution,  
          SignifThreshold = 0.05)

if("Global Test" %in% names(trait_AAOS_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_AAOS_presso$`MR-PRESSO results`)){
  if(is.character(trait_AAOS_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    aaos_snps <- trait_AAOS_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
  aaos_snps <- trait_AAOS.MRdat[trait_AAOS_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
  trait_AAOS.MRdat <- trait_AAOS.MRdat[-trait_AAOS_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_AAOS_presso$`MR-PRESSO results`)){
  aaos_presso <- trait_AAOS_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
    aaos_presso <- trait_AAOS_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ AAOS:** The MR-PRESSO global test for pleiotropy was `r if(aaos_presso < 0.05){paste('significant', ' (p = ', aaos_presso, ')', sep = "")} else {paste('non-significant')}`. `r if(aaos_presso < 0.05 & any(ls() %in% "aaos_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(aaos_snps, collapse=", ") )}`
<br> 

```{r pleiotropy ab42, echo = F, cache=T}
## ---- AB42 ---- ##
trait_AB42_presso <- mr_presso(BetaOutcome = "beta.outcome", 
          BetaExposure = "beta.exposure", 
          SdOutcome = "se.outcome", 
          SdExposure = "se.exposure", 
          OUTLIERtest = TRUE, 
          DISTORTIONtest = TRUE, 
          data = as.data.frame(trait_AB42.MRdat), 
          NbDistribution = NbDistribution,  
          SignifThreshold = 0.05)

if("Global Test" %in% names(trait_AB42_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_AB42_presso$`MR-PRESSO results`)){
  if(is.character(trait_AB42_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    ab42_snps <- trait_AB42_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
  ab42_snps <- trait_AB42.MRdat[trait_AB42_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
  trait_AB42.MRdat <- trait_AB42.MRdat[-trait_AB42_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_AB42_presso$`MR-PRESSO results`)){
  ab42_presso <- trait_AB42_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
    ab42_presso <- trait_AB42_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ AB42:** The MR-PRESSO global test for pleiotropy was `r if(ab42_presso < 0.05){paste('significant', ' (p = ', ab42_presso, ').', sep = "")} else {paste('non-significant')}`. `r if(ab42_presso < 0.05 & any(ls() %in% "ab42_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(ab42_snps, collapse=", ") )}`
<br> 

```{r pleiotropy ptau, echo = F, cache=T}
## ---- Ptau ---- ##
trait_Ptau_presso <- mr_presso(BetaOutcome = "beta.outcome", 
          BetaExposure = "beta.exposure", 
          SdOutcome = "se.outcome", 
          SdExposure = "se.exposure", 
          OUTLIERtest = TRUE, 
          DISTORTIONtest = TRUE, 
          data = as.data.frame(trait_Ptau.MRdat), 
          NbDistribution = NbDistribution,  
          SignifThreshold = 0.05)

if("Global Test" %in% names(trait_Ptau_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_Ptau_presso$`MR-PRESSO results`)){
  if(is.character(trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    ptau_snps <- trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
  ptau_snps <- trait_Ptau.MRdat[trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
  trait_Ptau.MRdat <- trait_Ptau.MRdat[-trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_Ptau_presso$`MR-PRESSO results`)){
  ptau_presso <- trait_Ptau_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
    ptau_presso <- trait_Ptau_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ Ptau:** The MR-PRESSO global test for pleiotropy was `r if(ptau_presso < 0.05){paste('significant', ' (p = ', ptau_presso, ').', sep = "")} else {paste('non-significant')}`. `r if(ptau_presso < 0.05 & any(ls() %in% "ptau_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(ptau_snps, collapse=", ") )}`
<br> 

```{r pleiotropy tau, echo = F, cache=T}
## ---- Tau ---- ##
trait_Tau_presso <- mr_presso(BetaOutcome = "beta.outcome", 
          BetaExposure = "beta.exposure", 
          SdOutcome = "se.outcome", 
          SdExposure = "se.exposure", 
          OUTLIERtest = TRUE, 
          DISTORTIONtest = TRUE, 
          data = as.data.frame(trait_Tau.MRdat), 
          NbDistribution = NbDistribution,  
          SignifThreshold = 0.05)

if("Global Test" %in% names(trait_Tau_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_Tau_presso$`MR-PRESSO results`)){
  if(is.character(trait_Tau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    tau_snps <- trait_Tau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
  tau_snps <- trait_Ptau.MRdat[trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
  trait_Ptau.MRdat <- trait_Ptau.MRdat[-trait_Ptau_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_Tau_presso$`MR-PRESSO results`)){
  tau_presso <- trait_Tau_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
    tau_presso <- trait_Tau_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ Tau:** The MR-PRESSO global test for pleiotropy was `r if(tau_presso < 0.05){paste('significant', ' (p = ', tau_presso, ').', sep = "")} else {paste('non-significant.')}`. `r if(tau_presso < 0.05 & any(ls() %in% "tau_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(tau_snps, collapse=", ") )}`
<br> 

```{r pleiotropy hipv, echo = F, cache=T}
## ---- hipv ---- ##
trait_hipv_presso <- mr_presso(BetaOutcome = "beta.outcome", 
                               BetaExposure = "beta.exposure", 
                               SdOutcome = "se.outcome", 
                               SdExposure = "se.exposure", 
                               OUTLIERtest = TRUE, 
                               DISTORTIONtest = TRUE, 
                               data = as.data.frame(trait_hipv.MRdat), 
                               NbDistribution = NbDistribution,  
                               SignifThreshold = 0.05)

if("Global Test" %in% names(trait_hipv_presso$`MR-PRESSO results`) & "Distortion Test" %in% names(trait_hipv_presso$`MR-PRESSO results`)){
  if(is.character(trait_hipv_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`)){
    hipv_snps <- trait_hipv_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`
  } else {
    hipv_snps <- trait_hipv.MRdat[trait_hipv_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`, ]$SNP
    trait_hipv.MRdat <- trait_hipv.MRdat[-trait_hipv_presso$`MR-PRESSO results`$`Distortion Test`$`Outliers Indices`,]
  }
}

if("Global Test" %in% names(trait_hipv_presso$`MR-PRESSO results`)){
  hipv_presso <- trait_hipv_presso$`MR-PRESSO results`$`Global Test`$Pvalue
} else {
  hipv_presso <- trait_hipv_presso$`MR-PRESSO results`$Pvalue
}

```

**`r params$trait.name` ~ hippocampal volume:** The MR-PRESSO global test for pleiotropy was `r if(hipv_presso < 0.05){paste('significant', ' (p = ', hipv_presso, ').', sep = "")}else{paste('non-significant')}` `r if(hipv_presso < 0.05 & any(ls() %in% "hipv_snps")){paste('The following SNPs were removed due to pleiotropy:', paste(hipv_snps, collapse=", ") )}`
<br> 
  
##  Mendelian Randomization Analysis 
To obtain an overall estimate of causal effect, the SNP-exposure (Major Depressive Disorder) and SNP-outcome coefficients (Alzheimer’s disease and Alzheimer's Age of Onset) were combined in 1) a random-effects meta-analysis using an inverse-variance weighted approach (IVW); 2) a Weighted Median approach; 3) and Egger Regression. IVW is equivalent to a weighted regression of the SNP-outcome coefficients on the SNP-exposure coefficients with the intercept constrained to zero. This method assumes that all variants are valid instrumental variables based on the Mendelian randomization assumptions. The causal estimate of the IVW analysis expresses the causal increase in the outcome (or log odds of the outcome for a binary outcome) per unit change in the exposure. Weighted median MR allows for 50% of the instrumental variables to be invalid. MR-Egger regression allows all the instrumental variables to be subject to direct effects (i.e. horizontal pleiotropy), with the intercept representing bias in the causal estimate due to pleiotropy and the slope representing the causal estimate.

### `r params$trait.name` ~ LOAD
```{r MR: Trait_LOAD, echo=F, warning=F, message = FALSE}

##  Single snp analysis and meta analysis
single.trait_ad <- mr_singlesnp(trait_LOAD.MRdat, 
                               single_method = 'mr_wald_ratio', 
                               all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

##  MR: IVW analysis
res.trait_ad <- mr(trait_LOAD.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

```
\

Figure 1 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and risk of LOAD.

```{r scatter_plot LOAD, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 1: Scatterplot of SNP effects for the association of Trait and LOAD'}

##  Scatter Plot
scatter.trait_ad <- mr_scatter_plot(res.trait_ad, trait_LOAD.MRdat)
scatter.trait_ad[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + 
  guides(col = guide_legend(nrow = 1)) + scale_colour_discrete() +
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in risk of \n Alzheimer\'s disease (log odds ratio)')  

```
\

Figure 2 and Table 1 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` on risk of LOAD.

```{r forrest_plot LOAD, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 2: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_ad <- mr_forest_plot(single.trait_ad)
forrest.trait_ad[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('log OR (95% CI) for AD per unit \n increase in', params$trait.name))

```
<br>

**Table 1:** MR estimates for `r params$trait.name` and LOAD
```{r MR LOAD table, echo=F}
as.data.frame(single.trait_ad %>% 
  select(SNP, b, se, p)) 

```
\

Figure 3 shows a funnel plot to detect pleiotropy and Table 2 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot LOAD, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 3: Funnel plot of the Trait – LOAD causal estimates against their precession'}
funnel.trait_ad <- mr_funnel_plot(single.trait_ad)
funnel.trait_ad[[1]] + theme_bw() + scale_colour_discrete() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12))

```
<br>

**Table 2:** Heterogenity tests for `r params$trait.name` and LOAD
```{r heterogeneity consumption_LOAD, echo=F, warning=F}
trait_ad.het <- mr_heterogeneity(trait_LOAD.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_ad.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 3:** Test for directional pleitropy for `r params$trait.name` and LOAD
```{r pleitropy test trait_load, echo=F}
trait_ad.plei <- mr_pleiotropy_test(trait_LOAD.MRdat)

as.data.frame(trait_ad.plei)

```

### `r params$trait.name` ~ AAOS

```{r MR: Trait ~ AAOS, echo=F, warning=F, message = FALSE}

single.trait_aaos <- mr_singlesnp(trait_AAOS.MRdat, 
                               single_method = 'mr_wald_ratio', 
                               all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

res.trait_aaaos <- mr(trait_AAOS.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))
```
\

Figure 4 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and AAOS.

```{r scatter_plot AAOS, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 4: Scatterplot of SNP effects for the association of Trait and AAOS'}

##  Scatter Plot
scatter.trait_aaos <- mr_scatter_plot(res.trait_aaaos, trait_AAOS.MRdat)
scatter.trait_aaos[[1]] + theme_bw() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + scale_colour_discrete() + 
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in risk of \n AAOS (HR)') 

```
\

Figure 5 and Table 4 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` on AAOS.

```{r forrest_plot AAOS, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 5: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_aaos <- mr_forest_plot(single.trait_aaos)
forrest.trait_aaos[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('HR (95% CI) for AAOS per unit \n increase in', params$trait.name))

```
<br>

**Table 4:** MR estimates for `r params$trait.name` and AAOS
```{r MR AAOS table, echo=F}
as.data.frame(single.trait_aaos %>% select(SNP, b, se, p))  

```
\

Figure 6 shows a funnel plot to detect pleiotropy and Table 5 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot AAOS, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 6:  Funnel plot of the traitohol Conumption – AAOS causal estimates against their precession'}
funnel.trait_aaos <- mr_funnel_plot(single.trait_aaos)
funnel.trait_aaos[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + scale_colour_discrete()

```
<br>

**Table 5:** Heterogenity tests for `r params$trait.name` and AAOS
```{r heterogeneity consumption_AAOS, echo=F, warning=F}
trait_aaos.het <- mr_heterogeneity(trait_AAOS.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_aaos.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 6:** Test for directional pleitropy for `r params$trait.name` and AAOS
```{r pleitropy test trait_aaos, echo=F}
trait_aaos.plei <- mr_pleiotropy_test(trait_AAOS.MRdat)

as.data.frame(trait_aaos.plei)

```

### `r params$trait.name` ~ AB42
```{r MR: trait ~ AB42, echo=F, warning=F, message = FALSE}

##  Single snp analysis and meta analysis
single.trait_ab <- mr_singlesnp(trait_AB42.MRdat, 
                               single_method = 'mr_wald_ratio', 
                               all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

##  MR: IVW analysis
res.trait_ab <- mr(trait_AB42.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

```
\

Figure 1 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and AB42.

```{r scatter_plot AB42, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 1: Scatterplot of SNP effects for the association of trait and AB42'}

##  Scatter Plot
scatter.trait_ab <- mr_scatter_plot(res.trait_ab, trait_AB42.MRdat)
scatter.trait_ab[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + 
  guides(col = guide_legend(nrow = 1)) + scale_colour_discrete() +
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in CSF \n AB42 levels (pg/mL)')  

```
\

Figure 2 and Table 7 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` CSF AB42 levels.

```{r forrest_plot AB42, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 2: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_ab <- mr_forest_plot(single.trait_ab)
forrest.trait_ab[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('pg/mL (95% CI) for CSF AB42 levels per \n unit increase in', params$trait.name))
```
<br>

**Table 7:** MR estimates for `r params$trait.name` and AB42
```{r MR AB42 table, echo=F}
as.data.frame(single.trait_ab %>% 
  select(SNP, b, se, p)) 

```
\

Figure 3 shows a funnel plot to detect pleiotropy and Table 8 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot AB42, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 3: Funnel plot of the trait – AB42 causal estimates against their precession'}
funnel.trait_ab <- mr_funnel_plot(single.trait_ab)
funnel.trait_ab[[1]] + theme_bw() + scale_colour_discrete() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12))

```
<br>

**Table 8:** Heterogenity tests for `r params$trait.name` and AB42
```{r heterogeneity traitumption_AB42, echo=F, warning=F}
trait_ab.het <- mr_heterogeneity(trait_AB42.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_ab.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 9:** Test for directional pleitropy for `r params$trait.name` and AB42
```{r pleitropy test trait_AB42, echo=F}
trait_ab.plei <- mr_pleiotropy_test(trait_AB42.MRdat)

as.data.frame(trait_ab.plei)

```

### `r params$trait.name` ~ Ptau
```{r MR: trait ~ Ptau, echo=F, warning=F, message = FALSE}

##  Single snp analysis and meta analysis
single.trait_ptau <- mr_singlesnp(trait_Ptau.MRdat, 
                               single_method = 'mr_wald_ratio', 
                               all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

##  MR: IVW analysis
res.trait_ptau <- mr(trait_Ptau.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

```
\

Figure 1 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and risk of Ptau.

```{r scatter_plot Ptau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 1: Scatterplot of SNP effects for the association of trait and Ptau'}

##  Scatter Plot
scatter.trait_ptau <- mr_scatter_plot(res.trait_ptau, trait_Ptau.MRdat)
scatter.trait_ptau[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + 
  guides(col = guide_legend(nrow = 1)) + scale_colour_discrete() +
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in CSF \n Ptau levels (pg/mL)')  

```
\

Figure 2 and Table 9 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` on risk of Ptau.

```{r forrest_plot Ptau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 2: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_ptau <- mr_forest_plot(single.trait_ptau)
forrest.trait_ptau[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('pg/mL (95% CI) for CSF Ptau levels per \n unit increase in', params$trait.name))

```
<br>

**Table 9:** MR estimates for `r params$trait.name` and Ptau
```{r MR Ptau table, echo=F}
as.data.frame(single.trait_ptau %>% 
  select(SNP, b, se, p)) 

```
\

Figure 3 shows a funnel plot to detect pleiotropy and Table 10 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot Ptau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 3: Funnel plot of the trait – Ptau causal estimates against their precession'}
funnel.trait_ptau <- mr_funnel_plot(single.trait_ptau)
funnel.trait_ptau[[1]] + theme_bw() + scale_colour_discrete() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12))

```
<br>

**Table 10:** Heterogenity tests for `r params$trait.name` and Ptau
```{r heterogeneity traitumption_Ptau, echo=F, warning=F}
trait_ptau.het <- mr_heterogeneity(trait_Ptau.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_ptau.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 11:** Test for directional pleitropy for `r params$trait.name` and Ptau
```{r pleitropy test trait_Ptau, echo=F}
trait_ptau.plei <- mr_pleiotropy_test(trait_Ptau.MRdat)

as.data.frame(trait_ptau.plei)

```


### `r params$trait.name` ~ Tau
```{r MR: trait ~ Tau, echo=F, warning=F, message = FALSE}

##  Single snp analysis and meta analysis
single.trait_tau <- mr_singlesnp(trait_Tau.MRdat, 
                               single_method = 'mr_wald_ratio', 
                               all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

##  MR: IVW analysis
res.trait_tau <- mr(trait_Tau.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

```
\

Figure 1 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and CSF Tau levels.

```{r scatter_plot Tau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 1: Scatterplot of SNP effects for the association of trait and Tau'}

##  Scatter Plot
scatter.trait_tau <- mr_scatter_plot(res.trait_tau, trait_Tau.MRdat)
scatter.trait_tau[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + 
  guides(col = guide_legend(nrow = 1)) + scale_colour_discrete() +
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in CSF \n Tau levels (pg/mL)')  

```
\

Figure 2 and Table 12 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` on risk of Tau.

```{r forrest_plot Tau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 2: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_tau <- mr_forest_plot(single.trait_tau)
forrest.trait_tau[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('pg/mL (95% CI) for CSF Tau levels per \n unit increase in', params$trait.name))

```
<br>

**Table 12:** MR estimates for `r params$trait.name` and Tau
```{r MR Tau table, echo=F}
as.data.frame(single.trait_tau %>% 
  select(SNP, b, se, p)) 

```
\

Figure 3 shows a funnel plot to detect pleiotropy and Table 13 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot Tau, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 3: Funnel plot of the trait – Tau causal estimates against their precession'}
funnel.trait_tau <- mr_funnel_plot(single.trait_tau)
funnel.trait_tau[[1]] + theme_bw() + scale_colour_discrete() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12))

```
<br>

**Table 14:** Heterogenity tests for `r params$trait.name` and Tau
```{r heterogeneity traitumption_Tau, echo=F, warning=F}
trait_tau.het <- mr_heterogeneity(trait_Tau.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_tau.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 15:** Test for directional pleitropy for `r params$trait.name` and Tau
```{r pleitropy test trait_Tau, echo=F}
trait_tau.plei <- mr_pleiotropy_test(trait_Tau.MRdat)

as.data.frame(trait_tau.plei)

```

### `r params$trait.name` ~ HV
```{r MR: trait ~ hipv, echo=F, warning=F, message = FALSE}

##  Single snp analysis and meta analysis
single.trait_hipv <- mr_singlesnp(trait_hipv.MRdat, 
                                 single_method = 'mr_wald_ratio', 
                                 all_method=c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

##  MR: IVW analysis
res.trait_hipv <- mr(trait_hipv.MRdat, method_list = c("mr_ivw_fe", "mr_weighted_median", "mr_egger_regression"))

```
\

Figure 1 illustrates the SNP-specific associations with `r params$trait.name` versus the association between each SNP and hippocampal volume.

```{r scatter_plot Hipv, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap= 'Fig. 1: Scatterplot of SNP effects for the association of trait and hippocampal volume'}

##  Scatter Plot
scatter.trait_hipv <- mr_scatter_plot(res.trait_hipv, trait_hipv.MRdat)
scatter.trait_hipv[[1]] + theme_bw() + theme(legend.position = "bottom", text = element_text(family="Times", size=12)) + 
  guides(col = guide_legend(nrow = 1)) + scale_colour_discrete() +
  labs(x = paste0('Per allele change in \n', params$trait.name), 
       y = 'Per allele change in \n hippocampal volume')  

```
\

Figure 2 and Table 12 shows the SNP-specific effects and overall IVW, weighted median and Egger regression causal estimates of genetically predicted `r params$trait.name` on risk of hippocampal volume.

```{r forrest_plot hipv, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap="Fig. 2: Forrest plot of Wald ratios and 95% CIs for SNP-specific and overall IVW, Weighted median and Egger associations"}

##  Forest Plot
forrest.trait_hipv <- mr_forest_plot(single.trait_hipv)
forrest.trait_hipv[[1]] + theme_classic() + 
  theme(legend.position = "none", text = element_text(family="Times", size=12)) + 
  labs(x = paste0('Hippocampal volume per \n unit increase in ', params$trait.name))

```
<br>
  
  **Table 12:** MR estimates for `r params$trait.name` and hippocampal volume
```{r MR hipv table, echo=F}
as.data.frame(single.trait_hipv %>% 
                select(SNP, b, se, p)) 

```
\

Figure 3 shows a funnel plot to detect pleiotropy and Table 13 show the results of Cochrans Q heterogeneity test to assess for the presence of pleiotropy. 

```{r funnel_plot hipv, echo=F, message = FALSE, warning=F, fig.width=7.5, fig.height=4.7, fig.align='center', fig.cap='Fig. 3: Funnel plot of the trait – hippocampal volume causal estimates against their precession'}
funnel.trait_hipv <- mr_funnel_plot(single.trait_hipv)
funnel.trait_hipv[[1]] + theme_bw() + scale_colour_discrete() + 
  theme(legend.position = "bottom", text = element_text(family="Times", size=12))

```
<br>
  
**Table 14:** Heterogenity tests for `r params$trait.name` and hippocampal volume
```{r heterogeneity trait_hipv, echo=F, warning=F}
trait_hipv.het <- mr_heterogeneity(trait_hipv.MRdat, method_list=c("mr_egger_regression", "mr_ivw"))

as.data.frame(trait_hipv.het %>% select(exposure, outcome, method, Q, Q_df, Q_pval))

```

**Table 15:** Test for directional pleitropy for `r params$trait.name` and hippocampal volume
```{r pleitropy test trait_hipv, echo=F}
trait_hipv.plei <- mr_pleiotropy_test(trait_hipv.MRdat)

as.data.frame(trait_hipv.plei)

```
<br>

## MR analysis results
```{r summary_results, echo=F, warning=F}

res_all <- res.trait_ad %>% 
  bind_rows(res.trait_aaaos) %>% 
  bind_rows(res.trait_ab) %>% 
  bind_rows(res.trait_ptau) %>% 
  bind_rows(res.trait_tau) %>% 
  bind_rows(res.trait_hipv) %>% 
  mutate(method = as.character(method)) %>%
  mutate(method = str_replace(method, "Inverse variance weighted \\(fixed effects\\)", 'IVW')) %>% 
  mutate(Signif = symnum(pval, corr = FALSE, na = FALSE, 
                  cutpoints = c(0, 0.001, 0.01, 0.05, 0.1, 1), 
                  symbols = c("***", "**", "*", ".", " "))) %>% 
  select(exposure, outcome, method, nsnp, b, se, pval, Signif)

res_all
```

```{r write_out_MR_results, echo=F}
write_csv(res_all, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_load_', params$p.threshold, '_MR_Results.csv'))
```






















