---
title: 'Mendelian Randomization: <trait> and Alzheimers disease'
author: "Shea Andrews"
date: "`r Sys.Date()`"
output:
  html_document:
    df_print: paged
    toc: true
    number_sections: true
    toc_depth: 4
    toc_float:
      collapsed: false
      smooth_scroll: false
editor_options:
  chunk_output_type: console
params:
  load.summary: 2_DerivedData/load_GWAS.Processed.gz
  aaos.summary: 2_DerivedData/aaos_GWAS.Processed.gz
  ab42.summary: 
  ptau.summary: 
  tau.summary: 
  hipv.summary: 
  trait.summary: 2_DerivedData/insom_GWAS.Processed.gz
  trait.clump: None
  trait.name: insom
  trait.blurb: 
  trait.code: insom
  out.path: 
  p.threshold: 5e-6
---

## Data sources
`r params$trait.blurb`

***Late Onset Alzheimer's disease*** [Lambert et al 2013](https://doi.org/10.1038/ng.2802): The International Genomics of Alzheimer’s Project (IGAP) is a meta-analysis of 4 previously published GWAS datasets: the European Alzheimer’s Disease Imitative (EADI), the Alzheimer Disease Genetics Consortium (ADGC), Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE), and Genetic and Environmental Risk in AD (GERAD) and includes a sample of 17,008 LOAD cases and 37,154 cognitively normal elder controls. Participants in IGAP were of European ancestry, the average age was 71 and 58.4% of participants were women.

***Alzheimer's Age of Onset Surivial*** [Huang et al 2017](https://doi.org/10.1038/nn.4587): A GWAS of age of onset in LOAD was conducted in 14,406 AD case samples and 25,849 control samples from the IGAP using Cox proportional hazard regressions. Participants were of European ancestry, in cases the the average AAO was 74.8 and 61.7% were women, in controls the average AAE was 79.0 and 59.6% were women.

***CSF Ab42, tau & ptau*** [Deming et al 2017](https://doi.org/10.1007/s00401-017-1685-y): A GWAS of CSF AB42, ptau and tau levels (pg/mL) was conducted in 3,146 participants. Participants were of Eurpean ancestry.

***Hippocampal Volume*** [Hibar et al 2017](https://doi.org/10.1038/ncomms13624): A GWAS of hippocampal volume perfomed in 26,814 (ENIGMA and CHARGE consortiums) individules of European Ancestry discovered 9 independent loci.


```{r setup, include=FALSE, echo=FALSE}
knitr::opts_chunk$set(echo = TRUE)

### ===== Load packages ===== ###

suppressMessages(library(tidyverse))   ## For data wrangling
suppressMessages(library(Hmisc))       ## Contains miscillaneous funtions
suppressMessages(library(TwoSampleMR)) ## For conducting MR https://mrcieu.github.io/TwoSampleMR/
suppressMessages(library(haploR))


##  ---- Find Proxy SNPs - uses haploR ---- ##
FindProxys <-function(snplist, dat){
  ProxySnps <- queryHaploreg(query = snplist, ldThresh = 0.8,
              ldPop = "EUR", timeout = 100, encoding = "UTF-8", verbose = TRUE) %>% 
  select(query_snp_rsid, rsID, chr, pos_hg38, r2, `D'`, is_query_snp, ref, alt) %>% 
  rename(ld.r2 = r2, Dprime = `D'`)

query_snps <- ProxySnps %>% filter(is_query_snp == 1) %>% select(query_snp_rsid, ref, alt)

proxy.snps <- left_join(ProxySnps, dat, by = c('rsID' = 'SNP')) %>% 
  filter(!is.na(Effect_allele)) %>%                       ## remove snps with missing information
  filter(is_query_snp == 0) %>%                           ## remove query snps
  filter(nchar(ref) == 1) %>% filter(nchar(alt) == 1) %>% ## remove indels
  group_by(query_snp_rsid) %>%                            ## by query snp
  arrange(-ld.r2) %>%                                     ## arrange by ld
  slice(1) %>%                                            ## select top proxy snp
  ungroup() %>% 
  rename(Effect_allele.proxy = Effect_allele, Non_Effect_allele.proxy = Non_Effect_allele) %>% ## rename alleles
  select(-chr, -pos_hg38, -is_query_snp) %>%              ## remove uneeded columns
  ## Join proxy snps to query snps
  left_join(query_snps, by = 'query_snp_rsid', suffix = c('.proxy', "")) %>% 
  ## for each proxy snp, coded correlated allels
  mutate(Effect_allele = ifelse(Effect_allele.proxy == ref.proxy, ref, alt)) %>%
  mutate(Non_Effect_allele = ifelse(Non_Effect_allele.proxy == ref.proxy, ref, alt)) %>% 
  rename(SNP = query_snp_rsid) 

proxy.snps
}

```

```{r datain, include=FALSE, echo=FALSE, cache=T}
### ===== Load Summary Statisitics ===== ###
## Assumes summayr statsticsts have the following column names:  
#   1. SNP           
#   2. CHR    
#   3. POS 
#   4. Effect_allele 
#   5. Non_Effect_allele   
#   6. EAF    
#   7. Beta     
#   8. SE     
#   9. P       
#   10. r2     
#   11. N
## ---- Trait ---- ##
trait.dat <- read_tsv(params$trait.summary) %>%
  filter(P < as.numeric(params$p.threshold))

## ---- Alzheimer's disease ---- ##
#  LOAD: Lambert et al 2013
igap.dat <- read_tsv(params$load.summary)
#  AAOS: Huang et al 2017
aaos.dat <- read_tsv(params$aaos.summary)

## ---- CSF ---- ##
#  csf: Demming et al 2017
ab42.dat <- read_tsv(params$ab42.summary)
ptau.dat <- read_tsv(params$ptau.summary)
tau.dat<- read_tsv(params$tau.summary)

## ---- Hippocampal Volume ---- ##
hipv.dat <- read_tsv(params$hipv.summary)
```

## Instrumental Variables
**LD Clumping:** For standard two sample MR it is important to ensure that the instruments for the exposure are independent. LD clumping can be performed using the data_clump function from the TwoSampleMR package, which uses EUR samples from the 1000 genomes project to estimate LD between SNPs and amonst SNPs that have and LD above a given threshold, only the SNP with the lowest p-value will be retained.
<br>

**Proxy SNPs:** SNPs associated with `r params$trait.name` were extracted from the GWAS of LOAD, AAOS, AB42, ptau and tau. Where SNPs were not available in the outcome GWAS, the EUR thousand genomes was queried to identified potential proxy SNPs that are in linkage disequilibrium (r2 > 0.8) of the missing SNP.

### `r params$trait.name`
```{r LD clumpting, echo=F, message=F, warning=F, cache=T}
Nsnps <- nrow(trait.dat)

if(params$trait.clump == 'None'){
  ## Convert to TwoSampleMR format to apply LD clumping
  mr_trait.dat <- format_data(trait.dat, type = 'exposure',
                               phenotype_col = 'trait',
                               snp_col = 'SNP',
                               beta_col = "Beta",
                               se_col = "SE",
                               eaf_col = "EAF",
                               effect_allele_col = "Effect_allele",
                               other_allele_col = "Non_Effect_allele",
                               pval_col = "P")
mr_trait.dat <- mutate(mr_trait.dat, exposure = 'trait')
## Clump 
mr_trait.dat_ld <- clump_data(mr_trait.dat, clump_r2 = 0.1, clump_kb = 250)

} else {
  ## Plink Pre-clumped
  mr_trait.dat_ld <- read_table2(params$trait.clump) %>% 
    filter(!is.na(CHR)) %>% 
    select(CHR, F, SNP, BP, P, TOTAL, NSIG)
}

#and filter
trait.dat <- trait.dat %>% filter(SNP %in% mr_trait.dat_ld$SNP)
```

`r params$trait.name`: `r Nsnps` SNPs (Table 1) were assoicated with were associated with `r params$trait.name` at p < `r params$p.threshold`. After LD clumping, `r Nsnps - nrow(trait.dat)` of `r Nsnps` SNPs were removed.
<br>

**Table 1:** Independent SNPS associated with `r params$trait.name`
```{r trait_table, echo=FALSE, message=F}
as.data.frame(trait.dat)
```
<br>

### LOAD
```{r data_joining trait_load, warning=FALSE, echo=FALSE, cache=T}
# LOAD
dat.load.trait <- igap.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.load.trait <- filter(dat.load.trait, is.na(CHR))

# If snps are missing, find proxy snp
if(nrow(miss.load.trait) >= 1){
  # Search for Proxy SNPs
  proxy.load.trait <- FindProxys(miss.load.trait$SNP, igap.dat)

  # Combine proxy snps with trait data
  dat.load.trait <- dat.load.trait %>%
    filter(SNP %nin% miss.load.trait$SNP) %>%
    bind_rows(select(proxy.load.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.load.trait)` were available in the LOAD GWAS (Table 2).


**Table 2:** SNPS associated with `r params$trait.name` avalible in LOAD GWAS
```{r trait_LOAD_table, echo=FALSE}
as.data.frame(dat.load.trait)
```
<br>

### AAOS

```{r data_joining trait_aaos, warning=FALSE, echo=FALSE, cache=T}
# AAOS
dat.aaos.trait <- aaos.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.aaos.trait <- filter(dat.aaos.trait, is.na(CHR))

# If snps are missing, find proxy snp
# If snps are missing, find proxy snp
if(nrow(miss.aaos.trait) >= 1){
  # Search for Proxy SNPs
  proxy.aaos.trait <- FindProxys(miss.aaos.trait$SNP, aaos.dat)
  
  # Combine proxy snps with trait data
  dat.aaos.trait <- dat.aaos.trait %>%
    filter(SNP %nin% miss.aaos.trait$SNP) %>%
    bind_rows(select(proxy.aaos.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.aaos.trait)` were available in the AAOS GWAS (Table 3).

**Table 3:** SNPS associated with <trait> available in AAOS GWAS
```{r trait_AAOS_table, echo=FALSE}
as.data.frame(dat.aaos.trait)
```
<br>

### AB42

```{r data_joining trait_ab42, warning=FALSE, echo=FALSE, cache=T}
# ab42
dat.ab42.trait <- ab42.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.ab42.trait <- filter(dat.ab42.trait, is.na(CHR))

# If snps are missing, find proxy snp
if(nrow(miss.ab42.trait) >= 1){
  # Search for Proxy SNPs
  proxy.ab42.trait <- FindProxys(miss.ab42.trait$SNP, ab42.dat)
  
  # Combine proxy snps with trait data
  dat.ab42.trait <- dat.ab42.trait %>%
    filter(SNP %nin% miss.ab42.trait$SNP) %>%
    bind_rows(select(proxy.ab42.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.ab42.trait)` were available in the CSF AB42 GWAS (Table 4).


**Table 4:** SNPS associated with `r params$trait.name` avalible in ab42 GWAS
```{r trait_ab42_table, echo=FALSE, cache=F}
as.data.frame(dat.ab42.trait)
```
<br>

### Ptau

```{r data_joining trait_ptau, warning=FALSE, echo=FALSE, cache=T}
# ptau
dat.ptau.trait <- ptau.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.ptau.trait <- filter(dat.ptau.trait, is.na(CHR))

# If snps are missing, find proxy snp
if(nrow(miss.ptau.trait) >= 1){
  # Search for Proxy SNPs
  proxy.ptau.trait <- FindProxys(miss.ptau.trait$SNP, ptau.dat)
  
  # Combine proxy snps with trait data
  dat.ptau.trait <- dat.ptau.trait %>%
    filter(SNP %nin% miss.ptau.trait$SNP) %>%
    bind_rows(select(proxy.ptau.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.ptau.trait)` were available in the CSF Ptau GWAS (Table 5).


**Table 5:** SNPS associated with `r params$trait.name` avalible in Ptau GWAS
```{r trait_ptau_table, echo=FALSE, cache=F}
as.data.frame(dat.ptau.trait)
```
<br>

### Tau

```{r data_joining trait_tau, warning=FALSE, echo=FALSE, cache=T}
# tau
dat.tau.trait <- tau.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.tau.trait <- filter(dat.tau.trait, is.na(CHR))

# If snps are missing, find proxy snp
if(nrow(miss.tau.trait) >= 1){
  # Search for Proxy SNPs
  proxy.tau.trait <- FindProxys(miss.tau.trait$SNP, tau.dat)
  
  # Combine proxy snps with trait data
  dat.tau.trait <- dat.tau.trait %>%
    filter(SNP %nin% miss.tau.trait$SNP) %>%
    bind_rows(select(proxy.tau.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.tau.trait)` were available in the CSF Tau GWAS (Table 6).


**Table 6:** SNPS associated with `r params$trait.name` avalible in tau GWAS
```{r trait_tau_table, echo=FALSE, cache=F}
as.data.frame(dat.tau.trait)
```
<br>

### Hippocampal volume

```{r data_joining trait_hipv, warning=FALSE, echo=FALSE, cache=T}
# hipv
dat.hipv.trait <- hipv.dat %>%
  right_join(select(trait.dat, SNP), by = 'SNP')

miss.hipv.trait <- filter(dat.hipv.trait, is.na(CHR))

# If snps are missing, find proxy snp
if(nrow(miss.hipv.trait) >= 1){
  # Search for Proxy SNPs
  proxy.hipv.trait <- FindProxys(miss.hipv.trait$SNP, hipv.dat)
  
  # Combine proxy snps with trait data
  dat.hipv.trait <- dat.hipv.trait %>%
    filter(SNP %nin% miss.hipv.trait$SNP) %>%
    bind_rows(select(proxy.hipv.trait, SNP, CHR, POS, Effect_allele, Non_Effect_allele, EAF, Beta, SE, P, r2, N)) %>%
    arrange(CHR, POS)
}
```

Of the the `r nrow(trait.dat)` SNPs associated with `r params$trait.name`, `r nrow(trait.dat) - nrow(miss.hipv.trait)` were available in the Hippocampal volume (Table 7).


**Table 7:** SNPS associated with `r params$trait.name` avalible in Hippocampal volume GWAS
```{r trait_hipv_table, echo=FALSE, cache=F}
as.data.frame(dat.hipv.trait)
```
<br>

## Data harmonization
Harmonize the exposure and outcome datasets so that the effect of a SNP on the exposure and the effect of that SNP on the outcome correspond to the same allele. The harmonise_data function from the TwoSampleMR package can be used to perform the harmonization step, by default it try's to infer the forward strand alleles using allele frequency information. EAF were not availbe in the IGAP summary statisitics, as such the allele frequencies reported in the AAOS anaylsis were used.
<br>

```{r format_trait, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
## ---- Exposure ---- ##
mr_trait.dat <- format_data(trait.dat, type = 'exposure',
                               phenotype_col = 'trait_cons',
                               snp_col = 'SNP',
                               beta_col = "Beta",
                               se_col = "SE",
                               eaf_col = "EAF",
                               effect_allele_col = "Effect_allele",
                               other_allele_col = "Non_Effect_allele",
                               pval_col = "P")
mr_trait.dat <- mutate(mr_trait.dat, exposure = params$trait.code)
```

### `r params$trait.name` ~ LOAD
```{r harmonize_trait_load, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
# Format LOAD
mr_load.trait.dat <- format_data(dat.load.trait, type = 'outcome',
                          phenotype_col = 'AD',
                          snp_col = 'SNP',
                          beta_col = "Beta",
                          se_col = "SE",
                          eaf_col = "EAF",
                          effect_allele_col = "Effect_allele",
                          other_allele_col = "Non_Effect_allele",
                          pval_col = "P")
mr_load.trait.dat <- mutate(mr_load.trait.dat, outcome = 'LOAD')

# harmonize LOAD
trait_LOAD.MRdat <- harmonise_data(mr_trait.dat, mr_load.trait.dat)
```

**Table 8:** Harmonized `r params$trait.name` and LOAD datasets
```{r trait_LOAD, echo=FALSE, cache=F}
as.data.frame(trait_LOAD.MRdat %>%
  select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

### `r params$trait.name` ~ AAOS
```{r harmonize_trait_aaos, warning=FALSE, echo=FALSE, include=FALSE, cache=T}

# Format AAOS
mr_aaos.trait.dat <- format_data(dat.aaos.trait, type = 'outcome',
                          phenotype_col = 'AAOS',
                          snp_col = 'SNP',
                          beta_col = "Beta",
                          se_col = "SE",
                          eaf_col = "EAF",
                          effect_allele_col = "Effect_allele",
                          other_allele_col = "Non_Effect_allele",
                          pval_col = "P")
mr_aaos.trait.dat <- mutate(mr_aaos.trait.dat, outcome = 'AAOS')

# harmonize AAOS
trait_AAOS.MRdat <- harmonise_data(mr_trait.dat, mr_aaos.trait.dat)
```

**Table 9:** Harmonized `r params$trait.name` and AAOS datasets
```{r trait_aaos, echo=FALSE, cache=F}
as.data.frame(trait_AAOS.MRdat %>%
  select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

### `r params$trait.name` ~ AB42
```{r harmonize_trait_ab42, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
# Format AB42
mr_ab42.trait.dat <- format_data(dat.ab42.trait, type = 'outcome',
                               phenotype_col = 'ab42',
                               snp_col = 'SNP',
                               beta_col = "Beta",
                               se_col = "SE",
                               eaf_col = "EAF",
                               effect_allele_col = "Effect_allele",
                               other_allele_col = "Non_Effect_allele",
                               pval_col = "P")
mr_ab42.trait.dat <- mutate(mr_ab42.trait.dat, outcome = 'ab42')

# harmonize AB42
trait_AB42.MRdat <- harmonise_data(mr_trait.dat, mr_ab42.trait.dat)

```

**Table 10:** Harmonized `r params$trait.name` and AB42 datasets
```{r trait_AB42, echo=FALSE, cache=F}
as.data.frame(trait_AB42.MRdat %>%
  select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

### `r params$trait.name` ~ Ptau
```{r harmonize_trait_ptau, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
# Format ptau
mr_ptau.trait.dat <- format_data(dat.ptau.trait, type = 'outcome',
                               phenotype_col = 'ptau',
                               snp_col = 'SNP',
                               beta_col = "Beta",
                               se_col = "SE",
                               eaf_col = "EAF",
                               effect_allele_col = "Effect_allele",
                               other_allele_col = "Non_Effect_allele",
                               pval_col = "P")
mr_ptau.trait.dat <- mutate(mr_ptau.trait.dat, outcome = 'ptau')

# harmonize ptau
trait_Ptau.MRdat <- harmonise_data(mr_trait.dat, mr_ptau.trait.dat)

```

**Table 11:** Harmonized `r params$trait.name` and Ptau datasets
```{r trait_Ptau, echo=FALSE, cache=F}
as.data.frame(trait_Ptau.MRdat %>%
  select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

### `r params$trait.name` ~ Tau
```{r harmonize_trait_tau, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
# Format tau
mr_tau.trait.dat <- format_data(dat.tau.trait, type = 'outcome',
                               phenotype_col = 'tau',
                               snp_col = 'SNP',
                               beta_col = "Beta",
                               se_col = "SE",
                               eaf_col = "EAF",
                               effect_allele_col = "Effect_allele",
                               other_allele_col = "Non_Effect_allele",
                               pval_col = "P")
mr_tau.trait.dat <- mutate(mr_tau.trait.dat, outcome = 'tau')

# harmonize tau
trait_Tau.MRdat <- harmonise_data(mr_trait.dat, mr_tau.trait.dat)

```

**Table 12:** Harmonized `r params$trait.name` and Tau datasets
```{r trait_Tau, echo=FALSE, cache=F}
as.data.frame(trait_Tau.MRdat %>%
  select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

### `r params$trait.name` ~ Hippocampal Volume
```{r harmonize_trait_hipv, warning=FALSE, echo=FALSE, include=FALSE, cache=T}
# Format hipv
mr_hipv.trait.dat <- format_data(dat.hipv.trait, type = 'outcome',
                                phenotype_col = 'hipv',
                                snp_col = 'SNP',
                                beta_col = "Beta",
                                se_col = "SE",
                                eaf_col = "EAF",
                                effect_allele_col = "Effect_allele",
                                other_allele_col = "Non_Effect_allele",
                                pval_col = "P")
mr_hipv.trait.dat <- mutate(mr_hipv.trait.dat, outcome = 'hipv')

# harmonize hipv
trait_hipv.MRdat <- harmonise_data(mr_trait.dat, mr_hipv.trait.dat)

```

**Table 13:** Harmonized `r params$trait.name` and Hippocampal Volume datasets
```{r trait_hipv, echo=FALSE, cache=F}
as.data.frame(trait_hipv.MRdat %>%
                select(SNP, effect_allele.exposure, other_allele.exposure, effect_allele.outcome, other_allele.outcome, beta.exposure, beta.outcome, eaf.exposure, eaf.outcome, se.outcome, pval.outcome, se.exposure, pval.exposure))
```
<br>

```{r drop_SummaryStats, echo=FALSE, message=F, cache=F}
## Remove AD summary statstics from memory
rm(igap.dat, aaos.dat, ab42.dat, ptau.dat, tau.dat, hipv.dat)
```

```{r write, echo=FALSE, message=F, warning=F, include=F}
write_csv(trait_LOAD.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_load_', params$p.threshold, '_MRdat.csv'))
write_csv(trait_AAOS.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_aaos_', params$p.threshold, '_MRdat.csv'))
write_csv(trait_AB42.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_ab42_', params$p.threshold, '_MRdat.csv'))
write_csv(trait_Ptau.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_ptau_', params$p.threshold, '_MRdat.csv'))
write_csv(trait_Tau.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_tau_', params$p.threshold, '_MRdat.csv'))
write_csv(trait_hipv.MRdat, paste0(params$out.path, params$trait.code, '/', params$trait.code, '_hipv_', params$p.threshold, '_MRdat.csv'))
```
\
